types/payment_voucher.go

```diff
@@ -54,13 +56,13 @@ type PaymentVoucher struct {
 func DecodeVoucher(voucherRaw string) (*PaymentVoucher, error) {
 	_, cborVoucher, err := multibase.Decode(voucherRaw)
 	if err != nil {
-		return nil, err
+		return nil, errors.New("voucher could not be decoded")
```

Comment:
error wrap to help with debugging, or just let the dev do it by hand?

Reply:
I considered it. The info that comes out of the decoder are is unlikely to be very helpful for debugging and is likely to sow confusion, so I opted not to wrap.